blazor/spreadsheet/cellrange.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: post
+title: Cell Range Management in Blazor Spreadsheet | Syncfusion
+description: Checkout and learn here about the cell range including cell formatting, autofill, and clear functionalities in the Syncfusion Blazor Spreadsheet component and more.
+platform: Blazor
+control: Spreadsheet
+documentation: ug
+---
+
+# Cell range in Blazor Spreadsheet component
+
+A group of cells in a sheet is known as cell range.
```

Comment:
A cell range is a group of selected cells in a Spreadsheet that you can work with together.